src/main/java/org/testeditor/fixture/host/net/HelperTool.java

```diff
@@ -0,0 +1,42 @@
+package org.testeditor.fixture.host.net;
+
+import java.io.File;
+
+import org.apache.commons.lang3.SystemUtils;
+
+/**
+ * The HelperTool is just a container for convenience methods to proof
+ * prerequisites for different testcases.
+ */
+public class HelperTool {
+
+    private static String pathS3270;
+
+    /**
+     * @return true if the path to s3270 driver is not null and available, false
+     *         otherwise.
+     */
+    public static boolean isS3270DriverPresent(String s3270Path) {
+        if (s3270Path != null && !s3270Path.isEmpty()) {
+            pathS3270 = s3270Path;
+            return new File(pathS3270).exists();
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * @return true if OS = WINDOWS, false otherwise
+     */
+    public static boolean isOsWindows() {
```

Comment:
There is already a  utility function for that. Why add it here? As I see it, it is only used in the test. No need to add it to production code then.